spec/controllers/ansible_credential_controller_spec.rb

```diff
@@ -70,5 +70,19 @@
         controller.send(:button)
       end
     end
+
+    context 'tagging one or more ansible repositories from nested list' do
+      let(:params) { {:pressed => "ansible_repository_tag"} }
```

Comment:
Where this params is being used?

Reply:
Check these lines:
https://github.com/ManageIQ/manageiq-ui-classic/pull/3721/files#diff-54b42f95ef23717853e22fb2c20e95feR35
https://github.com/ManageIQ/manageiq-ui-classic/pull/3721/files#diff-d4b472c8dab787b55f8c7219e39c37e3R37